### ISSUE TITLE
Subdomain not showing icons

### ISSUE THREAD

**Issue by optads** [FortAwesome/Font-Awesome#16559]
We host our clients landing pages on our own server and url then we use a CNAME to show the page on the clients domain.

An example is here - site designed and edited at sites.optads.com/h4a site displays on offer.weareintherapy.com (client domain)

The issue we have is whilst the site shows at the client end, the font awesome icons don't. So we are really stuck on how to fix it. You can see from the example above that font awesome icons are only showing on the url we use for creating and editing the pages. However on the client domain everything except for the font awesome icons is showing. Any help will be much appreciated as its driving us mad

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Please check your browser console for CORS errors

```
Access to font at 'https://sites.optads.com/h4a/wp-content/plugins/elementor/assets/lib/font-awesome/webfonts/fa-solid-900.woff2' from origin 'https://offer.weareintherapy.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

This is not related to font awesome

Hope it helps

Closing here

**Comment by optads** [comment]
What are you talking about? Of course its related to Font Awesome!!!!!!!1

[ ] [http://optads.com/]
Tom Bagshaw
Managing Director at Optads
E: tom@optads.com [mailto:tom@optads.com]    W: optads.com [http://optads.com/]
On 30/04/2020 17:55:05, Geremia Taglialatela <notifications@github.com> wrote:
Closed #16559 [https://github.com/FortAwesome/Font-Awesome/issues/16559].
